packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -263,43 +261,56 @@ export function RoutingFormResponsesTableContent() {
   const { t } = useLocale();
   const { filter } = useFilterContext();
```

Comment:
Now, we rely on `useFilterContext()` only for the date selectors. All the other parameters are managed below.